Makefile

```diff
@@ -84,6 +84,25 @@ run: ## start the development server using Docker
 	@$(COMPOSE_RUN) dockerize -wait tcp://db:5432 -timeout 60s
 .PHONY: run
 
+ngrok: ## start the development server using Docker through ngrok
+ngrok: \
```

Comment:
```suggestion
ngrok: run
```

Reply:
unfortunately, this will break the help grep call

Reply:
Which one ?
In my terminal I have this:

```
$ make
ngrok                                                  start the development server using Docker through ngrok
ngrok-apply                                            start the development server using Docker through ngrok and apply terraform plan
```

Reply:
Sorry, my mistake :facepalm: